src/components/Button/Button.js

```diff
@@ -94,6 +112,12 @@ Button.propTypes = {
    */
   children: PropTypes.node,
 
+  /**
+   * Specify how the button itself should be rendered.
+   * Make sure to apply all props to the root node and render children appropriately
+   */
+  render: PropTypes.func,
```

Comment:
this proptype isn't strictly accurate; it could be either a function (i.e. a stateless functional react component), a React.component class, or even a string. it's complicated

Reply:
@Zachary String is a fair point. Given `typeof class {}` yields `function`, probably `PropTypes.oneOfType([PropTypes.func, PropTypes.string])`?